client/js/libs/handlebars/localedate.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
```

Comment:
I'm fine having these in 2 different helpers, at least for now. When we start messing up with the text itself, then we can use one within the other, but there will still be 2 helpers, so 2 files is ok IMO.